CauldronMods/Controller/Heroes/Gargoyle/CharacterCards/GargoyleCharacterCardController.cs

```diff
@@ -10,33 +10,170 @@ namespace Cauldron.Gargoyle
 {
     public class GargoyleCharacterCardController : HeroCharacterCardController
     {
+        private const int REDUCE_DAMAGE_POWER = 1;
+        private const int REDUCE_DAMAGE_INCAP = 2;
+        private const int INCREASE_DAMAGE_POWER = 1;
+        private const int INCREASE_DAMAGE_INCAP = 2;
+        private const int NUMBER_USES_ONCE = 1;
+
         public GargoyleCharacterCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
         }
+
+        /*
+         * Power:
+         * "Select a target. Reduce the next damage it deals by 1. Increase the next damage Gargoyle deals by 1."
+         */
         public override IEnumerator UsePower(int index = 0)
         {
+            List<SelectTargetDecision> storedResults = new List<SelectTargetDecision>();
+            IEnumerable<Card> choices = base.FindCardsWhere(new LinqCardCriteria((Card c) => c.IsTarget && c.IsInPlayAndHasGameText));
+            ReduceDamageStatusEffect reduceDamageStatusEffect;
+            IncreaseDamageStatusEffect increaseDamageStatusEffect; 
+            IEnumerator coroutine;
+            Card selectedCard;
+
```

Comment:
PowerNumerals are needed here to support Harpy's Applied Numerology. In theory they should be used on all powers, Just In Case(tm), but in practice they probably only matter on innate powers due to the potential for Guise + Harpy shenanigans.
```int numDamageDecrease = GetPowerNumeral(0, REDUCE_DAMAGE_POWER);
int numDamageIncrease = GetPowerNumeral(1, INCREASE_DAMAGE_POWER);```